packages/shared/src/process/exitTask.ts

```diff
@@ -1,9 +1,10 @@
 import { close as finalizeLaunchDarkly } from "../launch-darkly/close";
 import { close as finalizeMixPanel } from "../mixpanel/close";
+import { logger } from "../logger";
 
 export type ExitTask = () => Promise<void>;
 
-export const exitTasks: ExitTask[] = [finalizeLaunchDarkly, finalizeMixPanel];
+export const exitTasks: ExitTask[] = [finalizeLaunchDarkly, finalizeMixPanel, logger.close];
```

Comment:
I don't think this is safe to do without binding the method to the instance.

Either:
```suggestion
export const exitTasks: ExitTask[] = [finalizeLaunchDarkly, finalizeMixPanel, () => logger.close()];
```

Or we'll need to change the `close` method to be defined as an error function on the `Logger` class.

Reply:
Oh right, JS and `this`.